router.go

```diff
@@ -63,16 +76,64 @@ func NewRouter(mongoURL, mongoDbName, backendConnectTimeout, backendHeaderTimeou
 	if err != nil {
 		return nil, err
 	}
+
+	mongoReadToOptime, err := bson.NewMongoTimestamp(time.Date(1970, time.January, 1, 0, 0, 0, 0, time.UTC), 1)
+	if err != nil {
+		return nil, err
+	}
+
 	logInfo("router: logging errors as JSON to", logFileName)
 
+	reloadChan := make(chan bool, 1)
 	rt = &Router{
 		mux:                   triemux.NewMux(),
 		mongoURL:              mongoURL,
 		mongoDbName:           mongoDbName,
 		backendConnectTimeout: beConnTimeout,
 		backendHeaderTimeout:  beHeaderTimeout,
+		mongoReadToOptime:     mongoReadToOptime,
 		logger:                l,
+		ReloadChan:            reloadChan,
 	}
+
+	go func(r chan bool) {
```

Comment:
Consider making this a (package-visible) method of Router rather than an nested, anonymous function - it's very much nontrivial and quite important, so I think it deserves a proper declaration.

Reply:
Fair point, though I would counter that with what I'm doing here is no different from the reload function that currently exists within _router_api.go_

Reply:
This is now done